ios/blade/Info.plist

```diff
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>1.0.3</string>
```

Comment:
Mismatch between gradle version and this? 🤔

Reply:
Should be 1.0.2